lib/cisco_node_utils/bgp.rb

```diff
@@ -524,6 +582,7 @@ def reconnect_interval=(seconds)
     end
 
     def default_reconnect_interval
+      return nil if platform == :ios_xr
```

Comment:
Looks like it from the readme:


When a feature or attribute is excluded in this way, attempting to call `config_get` or `config_set` on an excluded node will result in a `Cisco::UnsupportedError` being raised. Calling `config_get_default` on such a node will always return `nil`.

I'll remove it and retest.